packages/x-date-pickers/src/MultiSectionDigitalClock/tests/MultiSectionDigitalClock.test.tsx

```diff
@@ -105,4 +105,67 @@ describe('<MultiSectionDigitalClock />', () => {
       expect(onChange.lastCall.firstArg).toEqualDateTime(new Date(2019, 0, 1, 15, 30));
     });
   });
+
+  describe('Keyboard support', () => {
+    it('should move item focus up by 5 on PageUp press', () => {
+      const handleChange = spy();
+      render(<MultiSectionDigitalClock onChange={handleChange} />);
+      const hoursSectionListbox = screen.getAllByRole('listbox')[0]; // get only hour section
+      const hoursOptions = within(hoursSectionListbox).getAllByRole('option');
+      const lastOptionIndex = hoursOptions.length - 1;
+
+      hoursOptions[lastOptionIndex].focus();
```

Comment:
Have you considered reducing the specificity of the test?
It seems that we are "trying too hard". 🙈 
WDYT about making the rendered item `autofocus` or simply firing a `Tab` key event to move the focus into the first focusable item and then emitting `PageUp` or `PageDown` on the `document.activeElement` (just as `userEvent.keyboard` would do)? 🤔